### PR TITLE
Added suport for beeswax

### PR DESCRIPTION
**Pull request by kabilans**
Pull request for Beeswax honeypot, hpfeeds. Sorry it took a while.

### CODE REVIEW

**Comment by jt6211** [comment]
Where is the code for the beeswax project?

**Comment by kabilans** [comment]
https://github.com/iankronquist/beeswax

Its a docker based honeypot we made for our capstone project. It runs wordpress and sql containers and monitors them and reports any activity not ignored in the filter (configurable). It is also possible to run any container and write appropriate filters to extend this honeypot. Cheers!

File: geoloc.json [hunk 1]
```diff
@@ -2,21 +2,23 @@
     "HOST": "localhost",
     "PORT": 10000,
     "IDENT": "geoloc", 
-    "SECRET": "",
+    "SECRET": "379bdb6af644413ab6807479d06d96bf",
```
Comment (jt6211) [inline geoloc.json]: Can you remove this?  And you may need to change your secret if this was an accidental exposure.

**Comment by jt6211** [comment]
LGTM once secret is removed.

**Pull request closed without merging**